app/models/user.js

```diff
@@ -0,0 +1,30 @@
+module.exports = (sequelize, DataTypes) => {
+  const user = sequelize.define('user', {
+    id: {
+      type: DataTypes.INTEGER,
+      unique: true,
+      autoIncrement: true,
+      primaryKey: true,
+      allowNull: false
+    },
+    first_name: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    last_name: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    email: {
+      type: DataTypes.STRING,
+      unique: true,
+      allowNull: false
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false
```

Comment:
think if it is important to use the timestamp fields?

Reply:
timestamps fields added to model